float64[2] target
---
int8 errorValue
